sdk/servicebus/azure-servicebus/azure/servicebus/_common/utils.py

```diff
@@ -196,6 +205,25 @@ def transform_messages_to_sendable_if_needed(messages):
         except AttributeError:
             return messages
 
+def create_messages_from_dicts_if_needed(messages, message_type):
+    """
+    This method is used to convert dict representations
+    of messages and to a list of ServiceBusMessage objects or ServiceBusBatchMessage.
```

Comment:
In what case will we convert to the dict repr of messages into `ServiceBusMessageBatch` object? -- I didn't see there's any code doing that